### PR TITLE
cct updates

### PR DESCRIPTION
**Pull request by jimhshen**

### CODE REVIEW

File: 16_00_cct.Rmd [hunk 1]
```diff
@@ -264,22 +264,23 @@ Hugh Cole has been director of Policy and Strategy at the City of Cape Town sinc
...
+[Brendan Maughan-Brown](https://sites.google.com/site/bmaughanbrown/) is an interdisciplinary social scientist with expertise on: the uptake of HIV-prevention and treatment services; behavioral economics; and the social and behavioral determinants of HIV risk. He is a Chief Research Officer at the Southern Africa Labour and Development Research Unit, University of Cape Town.
 
 ## Acknowledgements {-}
 
 The authors thank Gordon Inggs, Yogini Jivanji, Craig Kesson, Kayleen Simpson, Carol Wright, Joshua Hawley, and the IDEA team at J-PAL for support of the collaboration and for constructive comments on the draft.
 
-[^cct6]: A summary of implementation activities associated with the Data Strategy is provided at http://osf.io/2a7ev. The document is not yet publicly available but will be provided at this same location when it becomes available.
+[^cct6]: A summary of implementation activities associated with the Data Strategy is provided at http://osf.io/2a7ev (accessed 2020-12-11). The document is not yet publicly available but will be provided at this same location when it becomes available.
```
Comment (larsvilhuber) [inline 16_00_cct.Rmd:273]: The document should be cited, not URL. The extra sentence should be stricken.

File: 16_00_cct.Rmd [hunk 2]
```diff
@@ -184,7 +184,7 @@ The authors assess each of the Five Safes in light of CCT's reworking of the dat
...
...rred. The Research Branch reviews the applications for appropriateness and alignment with the [Research Framework and research priorities](http://osf.io/2a7ev)[^cctaccessdate] (see section Institutional setup above). Currently, the review workflow has associated SOPs, though areas of improvement have been identified, especially with respect to integration with the entire data sharing process. For example, the application process could trigger determination of security levels (linked to tags in the metadata) and eventual access restrictions. The application form is being redesigned as a web-based form to make it more accessible to external researchers and to help reduce processing time. Domain experts, analysts, and lawyers may all review applications, and applications are reviewed in the order that they are received. Where necessary, the research application triggers the development of a DUA, as described under the section Legal framework for granting data access (above). In the future, a rule may be added that institutional review board (IRB) approval is needed before access is formally granted to academic researchers: a requirement that benefits from publicly accessible metadata.
```
Comment (larsvilhuber) [inline 16_00_cct.Rmd:187]: Cite the OSF repo.

File: 16_00_cct.Rmd [hunk 3]
```diff
@@ -168,7 +168,7 @@ The negotiation and establishment of bespoke legal agreements to underpin data a
...
 
-[CCT's Intellectual Property Policy](https://resource.capetown.gov.za/documentcentre/Documents/Bylaws%20and%20policies/Intellectual_Property_Policy.pdf) provides guidance on Intellectual Property (IP) from publicly funded research and development, which is subject to case-by-case review. For non-commercial research output, CCT does not typically exert any IP ownership. IP of the original and derived data (not including statistical outputs) is retained by CCT, and acknowledgement of data sources is required in any resulting publications or public materials.
+[CCT's Intellectual Property Policy](https://resource.capetown.gov.za/documentcentre/Documents/Bylaws%20and%20policies/Intellectual_Property_Policy.pdf)[^cctaccessdate] provides guidance on Intellectual Property (IP) from publicly funded research and development, which is subject to case-by-case review. For non-commercial research output, CCT does not typically exert any IP ownership. IP of the original and derived data (not including statistical outputs) is retained by CCT, and acknowledgement of data sources is required in any resulting publications or public materials.
```
Comment (larsvilhuber) [inline 16_00_cct.Rmd:171]: Not a website - a document, which should be in the citations.

File: 16_00_cct.Rmd [hunk 4]
```diff
@@ -108,7 +108,7 @@ if ( knitr::is_latex_output() ) {
...
+A major initiative to facilitate data use is the development and implementation of structured +metadata_standards| to be applied to all CCT data. CCT's Data Science team together with the Information and Knowledge Management department have developed standards comprising a minimal required set of [metadata elements](http://osf.io/2a7ev)[^cctaccessdate] along with additional, recommended elements. The minimal set of elements is similar to the +Dublin_Core| but with unique considerations for administrative data. The intent of the required minimal set is to minimize the burden on data stewards and to maintain quality metadata. Core fields include a summary of the data set, update frequency, data access rights, reasons for restriction, data format, spatial coverage, temporal coverage, and information about the source and stewardship of the data set.
```
Comment (larsvilhuber) [inline 16_00_cct.Rmd:111]: The OSF repo should be cited in the bibliography, and text modified to "required set of metadata elements (Cole et al, 2020)" (where the Cole et al stems from the [@Cole2020] citation.

**Review by larsvilhuber — CHANGES_REQUESTED** [review]
The CCT changes are not in order. Multiple documents are referenced as websites, when instead they should be cited (bib) as documents (techreport). Please correct both here and in LaTeX.

**Comment by larsvilhuber** [comment]
@jimhshen  see requested changes

File: 16_00_cct.Rmd (hunk 1 above)
Comment (jimhshen) [reply 16_00_cct.Rmd:273]: The summary they link to is actually duplicated in appendix A, so I'll change it to that.

File: 16_00_cct.Rmd (hunk 2 above)
Comment (jimhshen) [reply 16_00_cct.Rmd:187]: The document in the OSF repo is our appendix B, will edit to that.

File: 16_00_cct.Rmd (hunk 4 above)
Comment (jimhshen) [reply 16_00_cct.Rmd:111]: Likewise with the other OSF links, this is appendix C

**Comment by jimhshen** [comment]
Updated the citations

**Review by jimhshen — COMMENTED** [review]
made the changes

**Review by larsvilhuber — APPROVED** [review]

**Pull request merged**